### ISSUE TITLE
Blank plot pane

### ISSUE THREAD

**Issue by leifdenby** [julia-vscode/julia-vscode#2968]
When I try to create just a simple plot:

```julia
using Plots
x = 1:10; y = rand(10); 
plot(x,y)
```
The plot pane opens/adds a new plot correctly, but the pane is empty I don't see anything (see attached screenshot).

![vscode-julia-empty-plot-pane](https://user-images.githubusercontent.com/2405019/181066379-258bb721-9403-4419-b0fb-7913ff1598f6.png)

Am I do something incorrectly?
Thank you for any help in advance!

I'm using julia 1.8.0-rc1, but I've also tried with julia 1.7.3 and I had the same issue.

vscode version info:

```
Version: 1.69.2
Commit: 3b889b090b5ad5793f524b5d1d39fda662b96a2a
Date: 2022-07-18T16:14:10.636Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Linux x64 5.15.0-41-generic
```

And the julia extension is version v1.6.28

**Comment by pfitzseb** [comment]
Can't repro this. Try going to Help > Toggle Developer Tools and check for errors there.

**Comment by leifdenby** [comment]
> Can't repro this. Try going to Help > Toggle Developer Tools and check for errors there.

Thank you for your quick reply @pfitzseb! I tried opening up the "Developer Tools" as you suggested, but I found that the window didn't open when I clicked in the menu (or even when I used the keyboard shortcut ctrl+shift+I). That prompted me to think that maybe the issue was around how I had installed vscode as I used flatpak and I think it puts the executable in a sandbox.

So, I uninstalled vscode with flatpak and instead installed with snap (following the instructions here: https://phoenixnap.com/kb/install-vscode-ubuntu) and now the plots render correctly!

For future reference, neither the `com.visualstudio.code` or `com.visualstudio.code-oss` in flatpak work.

I'll close this issue now since it has been resolved for me
